AlexWang-16/alexwang-16.github.io
# Add open source contribution listing pages

Fixes #20 
# Purpose of update
To render open source contribution pages for firefox debugger and brave browser. The pages should include a link with date and the contribution, which links to the associated blog post.

# Change log
* Add layout for rendering open-source contribution listing
* Add firefox debugger listing data
* Add brave browser listing data
* refactor first-blog-post.md
* rename folder from "opensource" to "open-source"
* Add support for page specific CSS in default.html layout
* Fix footer support for returning to page section broken from previous refactoring

# Test Plan
1. Go to alexwang.ca
2. On the navbar, click on "open source".
3. Click on firefox debugger or brave browser link.

Repeat the same steps in mobile browser.

Expected result:
User should be redirected to alexwang.ca/open-source/firefox-debugger or alexwang.ca/open-source/brave-browser. He/she should see a list of open source contributions I've made along with the date. They should be able to click on the links and go to the blog post associated with the contribution.

# Additional notes
Layout refactoring including the first blog post by removing empty class attributes for improved conformance with HTML5 standards and best practices.
